python/feathub/feature_tables/sinks/kafka_sink.py

```diff
@@ -53,6 +53,9 @@ def __init__(
             {} if producer_properties is None else producer_properties
         )
 
+    def is_bounded(self) -> bool:
```

Comment:
After thinking about this more, I find it counter-intuitive to mark a sink as bounded or unbounded. This is because boundedness is only meaningful is the object is readable. But a sink is typically not readable, as it typically won't have the attributes (e.g. kafka consumer properties) needed when consuming the data.

How about we just overwrite this method in sink.py such that this method throws exception for all sinks, similar to Sink::get_bounded_view()?